plugins/storage/volume/storpool/src/main/java/org/apache/cloudstack/storage/datastore/util/StorPoolUtil.java

```diff
@@ -517,7 +517,13 @@ public static SpApiResponse volumeUpdate(final String name, final Long newSize,
         return POST("MultiCluster/VolumeUpdate/" + name, json, conn);
     }
 
-    public static SpApiResponse volumeUpdateTags(final String name, final String uuid, Long iops,
+    public static SpApiResponse volumeRemoveTags(String name, Map<String, String> tags, SpConnectionDesc conn) {
+        Map<String, Object> json = new HashMap<>();
```

Comment:
@slavkap Better to create empty tags map here, instead of passing it to indicate removal of tags in the method, otherwise it's like an update tags.

_Map<String, String> tags = StorPoolHelper.addStorPoolTags(null, "", null, "");_